README.md

```diff
@@ -1,2 +1,130 @@
-# viam-tflite-cpu
-the module for running tflite models on the cpu
+# [`tflite_cpu` module](https://app.viam.com/module/viam/tflite_cpu)
+
+This [module](https://docs.viam.com/registry/modular-resources/) implements the [`rdk:service:mlmodel` API](https://docs.viam.com/appendix/apis/services/ml/) in an `tflite_cpu` model.
+With this model, you can deploy [TensorFlow Lite](https://www.tensorflow.org/lite) model files as long as your models adhere to the [model requirements](#model-requirements).
+<!-- It is supported on any CPU and Linux, Raspbian, MacOS and Android machines. -->
+It is supported on any CPU and Linux, Raspbian and MacOS machines.
```

Comment:
@bhaney can you confirm that this is supported on raspbian machines and not yet supported on android?

Reply:
correct, not yet on android, but yes on all pi's